DQM/Integration/python/clients/l1tstage2emulator_dqm_sourceclient-live_cfg.py

```diff
@@ -65,7 +65,7 @@
 process.hltFatEventFilter = hltHighLevel.clone()
 process.hltFatEventFilter.throw = cms.bool(False)
 # HLT_Physics now has the event % 107 filter as well as L1FatEvents
-process.hltFatEventFilter.HLTPaths = cms.vstring('HLT_L1FatEvents_v*', 'HLT_Physics_v*')
+process.hltFatEventFilter.HLTPaths = cms.vstring('HLT_L1FatEvents_v*', 'HLT_Physics_v*', 'HLT_HIPhysics_v*')
```

Comment:
I think it would be better to change this in the `if (process.runType.getRunType() == process.runType.hi_run):` block.

Reply:
@fabiocos as far as I can see with the new commit this change was implemented.